server/api/src/rsSimpleQuery.cpp

```diff
@@ -87,3 +90,5 @@ rsSimpleQuery( rsComm_t *rsComm, simpleQueryInp_t *simpleQueryInp,
     }
     return status;
 }
+
+#pragma clang diagnostic pop // ignored "-Wdeprecated-declarations"
```

Comment:
why does this one get a comment?  the others didn't...

Reply:
I put a comment on this one because the `clang diagnostic push` occurs at the top of the file (there's a lot of deprecated stuff being used in here :p). The other `push`es and `pop`s were close enough to one another that they could be visually inspected very easily to understand what is happening.

That said, I'm happy to add comments to the others, or remove this comment

Reply:
all good, don't feel strongly - just noticed the difference.

Reply:
In that case, I am going to leave things how they are. I'm hoping the next time we look at any of this is when SimpleQuery gets removed.